shavar/lists.py

```diff
@@ -31,11 +31,17 @@ def includeme(config):
         for list_config_file in os.listdir(list_config_dir):
             if list_config_file.endswith(".ini"):
                 list_name = list_config_file[:-len(".ini")]
-                list_config = ConfigParser.ConfigParser()
-                list_config.readfp(open(
-                    os.path.join(list_config_dir, list_config_file)
-                ))
-                list_configs.append({'name': list_name, 'config': list_config})
+                try:
```

Comment:
In case a faulty `.ini` is added to the `list_config_dir`, we don't want to kill the whole process.